### PR TITLE
Documentation: Update Capella model, import and export T4C based models

### PR DESCRIPTION
**Pull request by MoritzWeber0**
Add documentation: 
- How to migrate TeamForCapella based models
- How to import TeamForCapella models into the persistent workspace
- How to export models to a TeamForCapella server
- How to link TeamForCapella repositories
- How to create, start, stop and delete TeamForCapella repositories.

### CODE REVIEW

**Comment by MoritzWeber0** [comment]
Hi @jamilraichouni. Did you already have the time to look into the PR or should I assign someone else?

File: docs/user/docs/settings/tools/capella/teamforcapella/export/export-to-t4c.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Export a Capella model your persistent workspace to a TeamForCapella server
+
+<!-- prettier-ignore -->
+!!! warning
+    To export a Capella model to a TeamForCapella, you need
```
Comment (dominik003) [inline docs/user/docs/settings/tools/capella/teamforcapella/export/export-to-t4c.md:10]: Either `... to a TeamForCapella server, ...` or `... to TeamForCapella, ...`

File: docs/user/docs/settings/tools/capella/teamforcapella/import/import-from-t4c.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   <!-- prettier-ignore -->
+    !!! hint
+        ![Project already exists in workspace](./project-already-exists-in-workspace.png)
+        If you get the message `A project with the same name already exists in your workspace`,
+        change the name in the `Local Project Name` field.
+        ![Change local project name](./change-local-project-name.png)
+
+1. Select `Import all images` during the next step (step does not exist on
+   Capella 5.x.x). ![Import all images](./import-all-images.png)
+1. Confirm with `Finish`. One should see a success message:
```
Comment (dominik003) [inline docs/user/docs/settings/tools/capella/teamforcapella/import/import-from-t4c.md:38]:
```suggestion
1. Confirm with `Finish`. You should see a success message:
```

File: docs/user/docs/settings/tools/capella/teamforcapella/repository-management.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# TeamForCapella repository management
+
+<!-- prettier-ignore -->
+!!! info
+    TeamForCapella repositories can only be managed by administrators.
+
+For all actions, you have to navigate to `Profile` > `Settings` >
+`TeamForCapella` (below `Model sources`). Select your instance. On the right
+side, you should see a card with the name `Manage T4C repositories`. In this
+card, you can see all existing repositories with the status.
```
Comment (dominik003) [inline docs/user/docs/settings/tools/capella/teamforcapella/repository-management.md:15]:
```suggestion
card, you can see all existing repositories with their status.
```

File: docs/user/docs/settings/tools/capella/teamforcapella/update.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# How to update a Capella model to a higher version (in TeamForCapella)
+
+<!-- prettier-ignore -->
+!!! warning
+    Please create updates (e.g. in a Git repository) before you start the update process.
```
Comment (dominik003) [inline docs/user/docs/settings/tools/capella/teamforcapella/update.md:10]:
```suggestion
    Please create updates (e.g. in a Git repository) before you start the update process.
```
```suggestion
    Please create backups (e.g. in a Git repository) before you start the update process.
```
I assume this shouldn't be _update_ here

**Review by dominik003 — APPROVED** [review]
LGTM! Just a few small adjustments, then it can be merged

**Pull request merged**